content/finding-command-help.md

```diff
@@ -0,0 +1,131 @@
+
+### --help
+
+`--help` prints out a shortened version of the man page, usually including a brief example and all options for a command. `--help` prints output to the terminal session without navigating away from the shell prompt.
+
+**Use case**: A user wants to quickly reference all options for a known command in the current Terminal session.
+
+**Usage**: Type a command + `--help`, then hit <kbd>Enter</kbd>. This example uses the `cd` command. Commands must be an exact match.
+
+```bash
+cd --help
+```
+![--help output](/images/finding-command-help/--help-output.png)
+
+### whatis
+
+`whatis` provides a very short description of a command's functionality.
+
+**Use case**: A user wants a very brief reminder of a known command's functionality printed in the current Terminal session.
+
+**Usage**: Type `whatis` + command, then hit <kbd>Enter</kbd>. This example uses the `mkdir` command. Commands must be an exact match.
+
+```bash
+whatis mkdir
+```
+![whatis output](/images/finding-command-help/what-output.png)
+
+### which & dpkg -S
+
+`which` provides the directory location of a command. The output of `which` can be passed into `dpkg -S` to perform a search of associated packages.
+
+**Use case**: A user is attempting to learn about commands included with a recently installed program, and would like to know which installed package provides a specific command  
+
+**Usage**: Type `which` + command, then pipe the output of `which` into `dpkg -S` using `xargs`. The `xargs` command allows users to pass the output of a command as standard input into another command. The below example searches for the path of the `libreoffice` command, and then searches for the program associated with that path.
+
+```bash
+which libreoffice | xargs dpkg -S
+```
+![which output](/images/finding-command-help/which-output.png)
+
+Users may see an error stating no matching path can be found. This occurs when `which` returns a symbolic link instead of the actual installation directory. Packages create symbolic links because the Terminal only recognizes executable commands from specific default and user defined directories. `dpkg` only knows the intended default installation path for a package, but may not know which package owns a symbolic link. Pass the path output from `which` into the `ls -alh` command to verify if an executable file is symbolically linked.
+
+```bash
+which shutdown | xargs ls -alh
+```
+![which ls](/images/finding-command-help/which-ls.png)
+
+The output indicates the package containing the `open` command is symbolically linked to `/bin/systemctl`. Now pass `/bin/systemctl` into the `dpkg -S` command to determine the package that owns this directory.
```

Comment:
You referred to an `open` command that is not referenced anywhere else in the article.